src/core/server/saved_objects/mappings/lib/get_root_properties_objects.test.ts

```diff
@@ -32,7 +32,7 @@ test(`returns single object with type === 'object'`, () => {
         type: 'object',
       },
     },
-  };
+  } as const;
```

Comment:
When TS infers a type for an object `{ type: 'keyword' }`, it ends up with ` { type: string }` since the object can be mutated. `as const` asserts that the object is immutable and the final type will be `{ type: "keyword" }`.

Reply:
we can do `const mappings: IndexMapping =`, but it found `as const` easier to understand 🤷‍♂️